### PR TITLE
BitsumLLC.ParkControl version 1.5.0

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28487)

### CODE REVIEW

**Comment by BajajAditya123** [comment]
/

File: manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: BitsumLLC.ParkControl
+PackageVersion: 1.5.0
```
Comment (OfficialEsco) [inline manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.installer.yaml:5]:
```suggestion
PackageVersion: 1.5.0.10
```

File: manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BitsumLLC.ParkControl
+PackageVersion: 1.5.0
```
Comment (OfficialEsco) [inline manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.5.0.10
```

File: manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: BitsumLLC.ParkControl
+PackageVersion: 1.5.0
```
Comment (OfficialEsco) [inline manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.yaml:5]:
```suggestion
PackageVersion: 1.5.0.10
```

File: manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BitsumLLC.ParkControl
+PackageVersion: 1.5.0
+PackageLocale: en-US
+Publisher: Bitsum LLC
```
Comment (OfficialEsco) [inline manifests/b/BitsumLLC/ParkControl/1.5.0/BitsumLLC.ParkControl.locale.en-US.yaml:7]:
```suggestion
Publisher: Bitsum
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
You also need to change the version folder name
I think you can do this in VScode Web pressing `.` inside this PR

**Comment by BajajAditya123** [comment]
ok

**Pull request closed without merging**